webrender/examples/blob.rs

```diff
@@ -10,18 +10,16 @@ extern crate webrender;
 extern crate webrender_traits;
 extern crate rayon;
 
-use gleam::gl;
+#[path="common/boilerplate.rs"]
+mod boilerplate;
+
 use rayon::ThreadPool;
 use rayon::Configuration as ThreadPoolConfig;
 use std::collections::HashMap;
 use std::collections::hash_map::Entry;
 use std::sync::Arc;
 use std::sync::mpsc::{channel, Sender, Receiver};
-use webrender_traits::{BlobImageData, BlobImageDescriptor, BlobImageError, BlobImageRenderer, BlobImageRequest};
-use webrender_traits::{BlobImageResult, TileOffset, ImageStore, ColorF, ColorU, Epoch};
-use webrender_traits::{DeviceUintSize, DeviceUintRect, LayoutPoint, LayoutRect, LayoutSize};
-use webrender_traits::{ImageData, ImageDescriptor, ImageFormat, ImageRendering, ImageKey, TileSize};
-use webrender_traits::{PipelineId, RasterizedBlobImage, TransformStyle};
+use webrender_traits::*;
```

Comment:
I believe this is generally non-idiomatic, but so tempting!
I'm thinking if something like `use webrender_traits as wt` for later prefixing with `wt` would make it more explicit?